CRM/Batch/Form/Entry.php

```diff
@@ -810,8 +777,6 @@ private function processMembership(&$params) {
         // end of contribution related section
 
         unset($value['membership_type']);
-        unset($value['membership_start_date']);
```

Comment:
moved down to the single membership section for clarity, since this does not relate to renewal & causes confusion